src/app/rundown-view/components/rundown-tooltips/piece-tooltip/piece-tooltip.component.html

```diff
@@ -0,0 +1,12 @@
+<div #tooltipHoverElement (mouseenter)="emitNewHoverMouseEvent($event)" (mousemove)="emitNewHoverMouseEvent($event)" (mouseleave)="emitNewHoverMouseEvent()" class="c-hover-scrub-hover-element"></div>
```

Comment:
Just as for the MiniShelfTooltipComponent you can utilize `HostListener`
![image](https://github.com/tv2/ng-sofie/assets/23049647/6a87311c-ecb9-453f-92d3-81c61b23d3b2)

Reply:
@LindvedKrvang We can, but I don't see any benefit of doing that, instead emit event on to the element, or I miss something?

Reply:
They do almost the same. Main differnce is that you don't need to call the same method three times in the html (from the same place)

Reply:
Done